cypress/integration/common/index.js

```diff
@@ -51,6 +51,21 @@ Given('user is on landing Genetics Terms page', () => {
 	cy.visit(baseURL);
 });
 
+
+Given('user enters {string} as the URL', (a) => {
+	cy.visit(a);
+});
+Then('the system redirects user to {string}',(a) =>{
+	
+	cy.location('pathname').should('eq','/def/a33');
+
+});
+And('the system appends {string} to the URL', (a) => {
+	cy.location('search').should('eq','?redirect=true')
```

Comment:
This is a valid step - and should be the only one you need, since you are simply adding this step in PrettyUrlRedirect.feature

Reply:
You are passing a string parameter 'a' - which equals to _?redirect=true_ . Instead of hardcoding it, you should use the passed value of _a_ and your code will look like :  `cy.location('search').should('eq',a);`

Reply:
ok fixed it